test/unit/object/test_events.py

```diff
@@ -181,6 +182,18 @@ def mock_event():
 
 
 @pytest.fixture()
+def mock_event_object(mock_box_session):
```

Comment:
consider renaming the mock_event fixture above to mock_event_json or something like that... just for clarity.